HermesProxy/World/Client/PacketHandlers/ItemHandler.cs

```diff
@@ -214,5 +214,17 @@ void HandleItemEnchantTimeUpdate(WorldPacket packet)
             enchant.OwnerGuid = packet.ReadGuid().To128(GetSession().GameState);
             SendPacketToClient(enchant);
         }
+
+        [PacketHandler(Opcode.SMSG_ENCHANTMENT_LOG)]
+        void HandleEnchantmentLog(WorldPacket packet)
+        {
+            EnchantmentLog enchantment = new EnchantmentLog();
+            enchantment.Caster = packet.ReadPackedGuid().To128(GetSession().GameState);
+            enchantment.Owner = enchantment.Caster;
+            enchantment.ItemGUID = packet.ReadPackedGuid().To128(GetSession().GameState);
+            enchantment.ItemID = (Int32)packet.ReadUInt32();
+            enchantment.Enchantment = (Int32)packet.ReadUInt32();
```

Comment:
Otherwise:
```suggestion
            enchantment.ItemID = (int)packet.ReadUInt32();
            enchantment.Enchantment = (int)packet.ReadUInt32();
```